MakerTracker/DBModels/Profile.cs

```diff
@@ -1,47 +1,29 @@
 using System;
 using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
 
 namespace MakerTracker.DBModels
 {
     public class Profile
     {
         public int Id { get; set; }
         public string Auth0Id { get; set; }
-
-        [Display(Name = "First Name")]
+        public string CompanyName { get; set; }
         public string FirstName { get; set; }
```

Comment:
I'm thinking we want to mark these up with `[MaxLength]` and reflect that also in the UI forms.

Reply:
That's a good point . I did strip these attributes off as I would like to have (at least eventually) everything coming in/out of the api to be a DTO .

Reply:
`[MaxLength]` affects the database migration - ie. `nvarchar(50)` as opposed to `nvarchar(MAX)`